### PR TITLE
doc: add dedicated machines private location documentation, closes HYB-411

### PR DESCRIPTION
**Pull request by tpetillot**
[doc: add reference to control plane configuration page](https://github.com/gatling/frontline-cloud-doc/commit/119229077837ecb6c29021984a0fcde31c8d25cb)

[doc: control plane installation using docker](https://github.com/gatling/frontline-cloud-doc/commit/b1cfeca43fda8855c67fea58d9b673b83aadc987)

[doc: rename with provider only control plane installations](https://github.com/gatling/frontline-cloud-doc/commit/33d034b0ac368a0acb5cd7bb0aa05386803c059c)

[doc: add dedicated machines private location documentation, closes HYB-411](https://github.com/gatling/frontline-cloud-doc/commit/31dac9801b238b60378c714d6fe20348c0b8e403) 

[doc: remove system property documentation for control plane token](https://github.com/gatling/frontline-cloud-doc/commit/7bf73630fdd08a845f37af04ba2bc5272ce7e51f) 

**Motivation:**
Only confusing, we don't document how to pass system properties to control plane, and it may confuse users with "system-properties" configuration available on locations.

### CODE REVIEW

File: content/reference/admin/private_locations/installation/docker/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+version: '3.8'
+services:
+  control-plane:
+    image: gatlingcorp/control-plane:latest
+    container_name: control-plane
+    volumes:
+      - /etc/control-plane:/app/conf
+```
+
+In order for the `/etc/control-plane` directory to function properly, it must include a file named `control-plane.conf`.
```
Comment (Obero) [inline content/reference/admin/private_locations/installation/docker/index.md:33]:
```suggestion
The `/etc/control-plane` directory must include a file named `control-plane.conf`. 
```

File: content/reference/admin/private_locations/installation/docker/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+For instance, if you are using AWS, you can mount the configuration and credentials files from your home directory by including the following lines in your YAML configuration:
+```yaml
+volumes:
+  - /etc/control-plane:/app/conf
+  - ~/.aws:/root/.aws
+```
+{{< /alert >}}
+
+{{< alert info >}}
+When using private packages, you will need to expose the port by including the following lines in your YAML configuration:
```
Comment (Obero) [inline content/reference/admin/private_locations/installation/docker/index.md:50]: Specify which configuration file maybe ?

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+To do this, follow these steps:
+
+1. Create the authorized_keys file in the gatling/.ssh directory on the host:
+```bash
+touch gatling/.ssh/authorized_keys
+chmod 600 gatling/.ssh/authorized_keys
+```
+
+2. Add the public key to gatling/.ssh/authorized_keys. You can do this either by using the ssh-copy-id command:
```
Comment (Obero) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:118]:
```suggestion
2. Add the public key to gatling/.ssh/authorized_keys on the host. You can do this either by using the ssh-copy-id command:
```

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+```bash
+ssh-copy-id -i control-plane.pub gatling@host
+```
+Or, manually by appending the public key to the file with:
+```bash
+cat control-plane.pub >> gatling/.ssh/authorized_keys
+```
+
+3. Add the private key to the control plane
```
Comment (Obero) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:127]:
```suggestion
3. Add the private key to the control plane. Refer to your control plane config file for the expected private key file path: for your location, the value of `ssh.private-key.path`.
```

**Review by Obero — COMMENTED** [review]
Nice PR !

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+lastmod: 2023-10-13T08:10:39+00:00
+weight: 22055
+---
+
+## Dedicated Machines
+
+To use Dedicated Machines private locations, the control plane must have access to your dedicated machines.
+
+To set up a dedicated location configuration, ensure that each host is reachable through SSH on a specific port.
+The control plane possesses a private key for establishing connections, while the corresponding public key is shared and configured on every host in the location.
```
Comment (Isammoc) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:15]: An introduction paragraph?

```suggestion
## Dedicated Machines

You can configure private locations with already existing servers. The control plane will be able to use configured dedicated machines as load generator during your simulations.

The control plane must have access to your dedicated machines. Ensure that each host is reachable through SSH on a specific port.
The control plane possesses a private key for establishing connections, while the corresponding public key is shared and configured on every host in the location.
```

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+      # Private location type
+      type = "dedicated"
+      ssh {
+        # SSH user
+        user = "gatling"
+        ## SSH private key
+        private-key {
+          # SSH private key path used to secure connections
+          path = "private/key/path"
+          ## SSH private key password (optional, default: none)
```
Comment (Isammoc) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:53]:
```suggestion
          # SSH private key password (optional, default: none)
```

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+          password = "password"
+           # SSH private key password with an environment variable
+          password = ${?PRIVATE_KEY_PASSWORD}
+        }
+        # SSH port (optional, default: 22)
+        # port = 22
+        # SSH connection timeout (optional, default 15 seconds)
+        # connection-timeout = 15 seconds
```
Comment (Isammoc) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:61]: Should we have a convention that comment on commented code should be commented? :thinking: 

```suggestion
        # # SSH port (optional, default: 22)
        # port = 22
        # # SSH connection timeout (optional, default 15 seconds)
        # connection-timeout = 15 seconds
```

or:

```suggestion
        ## SSH port (optional, default: 22)
        # port = 22
        ## SSH connection timeout (optional, default 15 seconds)
        # connection-timeout = 15 seconds
```

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+      }
+      # Working directory on hosts, must be executable (optional, default: /tmp)
+      # working-directory = "/tmp
```
Comment (Isammoc) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:64]:
```suggestion
      ## Working directory on hosts, must be executable (optional, default: /tmp)
      # working-directory = "/tmp
```

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+      hosts = [
+        "localhost",
+        "0.0.0.0"
+      ]
+      
+      # Java configuration (following configuration properties are optional)
+      # System properties (optional)
+      system-properties {
+        # ExampleKey = ExampleValue
+      }
```
Comment (Isammoc) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:76]:
```suggestion
      ## Java configuration (following configuration properties are optional)
      ## System properties (optional)
      # system-properties {
        ## ExampleKey = ExampleValue
      #}
```

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+      # [
+      #   "-Xmx4G", 
+      #   "-XX:MaxInlineLevel=20", 
+      #   "-XX:MaxTrivialSize=12", 
+      #   "-XX:+IgnoreUnrecognizedVMOptions", 
+      #   "--add-opens=java.base/java.nio=ALL-UNNAMED", 
+      #   "--add-opens=java.base/jdk.internal.misc=ALL-UNNAMED"
+      # ]
+      # Based on your instance configuration, you may want to update Xmx and Xms values.
+      # jvm-options = ["-Xmx4G", "-Xms512M"]
```
Comment (Isammoc) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:90]:
```suggestion
      ## Overwrite JAVA_HOME definition (optional)
      # java-home = "/usr/lib/jvm/zulu"
      ## JVM Options (optional)
      ## Default ones, that can be overriden with precedence:
      ## [
      ##   "-Xmx4G", 
      ##   "-XX:MaxInlineLevel=20", 
      ##   "-XX:MaxTrivialSize=12", 
      ##   "-XX:+IgnoreUnrecognizedVMOptions", 
      ##   "--add-opens=java.base/java.nio=ALL-UNNAMED", 
      ##   "--add-opens=java.base/jdk.internal.misc=ALL-UNNAMED"
      ## ]
      ## Based on your instance configuration, you may want to update Xmx and Xms values.
      # jvm-options = ["-Xmx4G", "-Xms512M"]
```

But as the default values are the one commented by default...

```suggestion
      ## Overwrite JAVA_HOME definition (optional)
      # java-home = "/usr/lib/jvm/zulu"

      ## JVM Options (optional)
      ## Based on your instance configuration, you may want to update Xmx and Xms values.
      # jvm-options = [
      #   "-Xmx4G", 
      #   "-XX:MaxInlineLevel=20", 
      #   "-XX:MaxTrivialSize=12", 
      #   "-XX:+IgnoreUnrecognizedVMOptions", 
      #   "--add-opens=java.base/java.nio=ALL-UNNAMED", 
      #   "--add-opens=java.base/jdk.internal.misc=ALL-UNNAMED"
      # ]
```

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+
+Next, you'll need to configure the authorized keys on the host for a user named, for the example, `gatling`.
+
+To do this, follow these steps:
+
+1. Create the authorized_keys file in the gatling/.ssh directory on the host:
+```bash
+touch gatling/.ssh/authorized_keys
+chmod 600 gatling/.ssh/authorized_keys
+```
```
Comment (Isammoc) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:116]: I guess the code block is part of the item in the list.

````suggestion
1. Create the authorized_keys file in the gatling/.ssh directory on the host:
    ```bash
    touch gatling/.ssh/authorized_keys
    chmod 600 gatling/.ssh/authorized_keys
    ```
````

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+
+2. Add the public key to gatling/.ssh/authorized_keys. You can do this either by using the ssh-copy-id command:
+```bash
+ssh-copy-id -i control-plane.pub gatling@host
+```
+Or, manually by appending the public key to the file with:
+```bash
+cat control-plane.pub >> gatling/.ssh/authorized_keys
+```
```
Comment (Isammoc) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:125]:
````suggestion
    ```bash
    ssh-copy-id -i control-plane.pub gatling@host
    ```

    Or, manually by appending the public key to the file with:

    ```bash
    cat control-plane.pub >> gatling/.ssh/authorized_keys
    ```
````

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 13]
```diff
@@ -0,0 +1,147 @@
...
+    type = "dedicated"
+    ...
+    debug {
+      keep-load-generator-alive = true
+    }
+  }
+]
+```
+
+Afterward, you can examine the logs via systemd logs using the identifier 'gatling' with `journalctl --identifier gatling`.
```
Comment (Isammoc) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:147]:
```suggestion
Afterward, you can examine the logs via systemd logs using the identifier '`gatling`' with `journalctl --identifier gatling`.
```

File: content/reference/admin/private_locations/configuration/dedicated/index.md (hunk 10 above)
Comment (tpetillot) [reply content/reference/admin/private_locations/configuration/dedicated/index.md:90]: Maybe one day, but not today (must update all related documentations)

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 14]
```diff
@@ -0,0 +1,149 @@
...
+description: "Deploy Load Generators on dedicated machines that you manage."
+lead: "Deploy Load Generators on dedicated machines that you manage."
+date: 2023-01-12T16:46:04+00:00
+lastmod: 2023-10-13T08:10:39+00:00
+weight: 22055
+---
+
+## Dedicated Machines
+
+You can configure private locations with already existing servers.
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:12]:
```suggestion
You can configure private locations with pre-existing servers. 
```

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 15]
```diff
@@ -0,0 +1,149 @@
...
+lead: "Deploy Load Generators on dedicated machines that you manage."
+date: 2023-01-12T16:46:04+00:00
+lastmod: 2023-10-13T08:10:39+00:00
+weight: 22055
+---
+
+## Dedicated Machines
+
+You can configure private locations with already existing servers. 
+The control plane will be able to use configured dedicated machines as load generator during your simulations.
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:13]:
```suggestion
The control plane will be able to use configured dedicated machines as load generators during your simulations.
```

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 16]
```diff
@@ -0,0 +1,149 @@
...
+
+The control plane must have access to your dedicated machines. Ensure that each host is reachable through SSH on a specific port.
+
+The control plane possesses a private key for establishing connections, while the corresponding public key is shared and configured on every host in the location.
+
+### Host prerequisites
+
+Location hosts environment must include:
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:21]:
```suggestion
The location host environment must include:
```

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 17]
```diff
@@ -0,0 +1,149 @@
...
+- Java runtime environment
+- `bash` the GNU Project's shell
+- `jq` a lightweight and flexible command-line JSON processor
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:24]: Should we provide external links to any of these?

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 18]
```diff
@@ -0,0 +1,149 @@
...
+- `curl` command line tool for transferring data with URLs
+- `~/.ssh/authorized_keys` with control plane public key
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:26]:
```suggestion
- `~/.ssh/authorized_keys` with a control plane public key
```

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 19]
```diff
@@ -0,0 +1,149 @@
...
+  ]
+}
+```
+
+{{< alert warning >}}
+Working directory must be executable
+{{< /alert >}}
+
+## Troubleshooting
+
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:103]:
```suggestion
This section contains tips for troubleshooting common issues with dedicated machines. If you have an issue not covered here, consider posting a question on the[ Community Forum](https://community.gatling.io/c/enterprise/6). 
```

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 20]
```diff
@@ -0,0 +1,149 @@
...
+### SSH keys
+
+Execute the command `ssh-keygen -t ed25519 -f control-plane` to generate your key pair. _(You have the option to configure a password during this process)_
+
+This will generate the `control-plane` private key and `control-plane.pub` public key in the current directory where the command was executed.
+
+Next, you'll need to configure the authorized keys on the host for a user named, for the example, `gatling`.
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:110]:
```suggestion
Next, you'll need to configure the authorized keys on the host for a user named, for example, `gatling`.
```

Comment (stb13579) [reply content/reference/admin/private_locations/configuration/dedicated/index.md:110]: I was a little confused by this, but I think removing 'the' fixes it.

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 21]
```diff
@@ -0,0 +1,149 @@
...
+
+To do this, follow these steps:
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:112]:
```suggestion
To configure the authorized keys on the host:
```

Comment (stb13579) [reply content/reference/admin/private_locations/configuration/dedicated/index.md:112]: Here you can omit the 'follow these steps' but explicitly say what it is for.

File: content/reference/admin/private_locations/configuration/dedicated/index.md [hunk 22]
```diff
@@ -0,0 +1,149 @@
...
+  ```bash
+  ssh-copy-id -i control-plane.pub gatling@host
+  ```
+  Or, manually by appending the public key to the file with:
+  ```bash
+  cat control-plane.pub >> gatling/.ssh/authorized_keys
+  ```
+
+3. Add the private key to the control plane. Refer to your control plane config file for the expected private key file path: for your location, the value of `ssh.private-key.path`.
+
```
Comment (stb13579) [inline content/reference/admin/private_locations/configuration/dedicated/index.md:130]: This is a perfectly written procedure!😀

File: content/reference/admin/private_locations/installation/azure/index.md [hunk 23]
```diff
@@ -49,7 +49,7 @@ Write down the storage account name, you will need it in further steps.
 
 Once again, refer to the [Azure File Share documentation](https://learn.microsoft.com/en-us/azure/storage/files/storage-how-to-use-files-portal?tabs=azure-portal) for details.
 
-In your storage account, let's create a File Share for the control plane configuration file:
+In your storage account, let's create a File Share for the [Control Plane configuration]({{< ref "../../configuration" >}}) file:
 * Access to your storage account page
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/azure/index.md:53]: If this is a procedure that should be followed sequentially it is better to use a numbered list than an unordered list.

**Review by stb13579 — APPROVED** [review]
@tpetillot great work! I made a few suggestions to improve the text. If you have any questions or don't understand my comments don't hesitate to ask me.

**Pull request merged**